lib/ssl/doc/src/ssl.xml

```diff
@@ -1424,10 +1450,17 @@ fun(srp, Username :: string(), UserState :: term()) ->
       <name since="OTP 20.3" name="filter_cipher_suites" arity="2" />
       <fsummary></fsummary>
       <desc><p>Removes cipher suites if any of the filter functions
-      returns false for any part of the cipher suite. This function
-      also calls default filter functions to make sure the cipher
-      suites are supported by crypto. If no filter function is supplied for some
-      part the default behaviour is fun(Algorithm) -> true.</p>
+      returns false for any part of the cipher suite. If no filter function is supplied for some
+      part the default behaviour regards it as if there was a filter function that returned true.
+      
+      For examples see <seealso marker="ssl:using_ssl#Customizing-cipher-suits"> Customizing cipher suits </seealso> 
+      
+      Additionaly this function also filters the cipher suites to
+      exclude cipher suites not supported by the cryptolib used by the
+      OTP crypto application.  That is calling
```

Comment:
```suggestion
      OTP crypto application. That is calling
```

Reply:
Even the nitpick ones. It looks better without the extra white space :)